README.md

```diff
@@ -284,6 +381,10 @@ Cursors are used to add paging functionality to certain nodes, like friends list
 
 ![image cursors](./images/Cursors.png)
 
+# Working with Logs
+
+TODO
+
 # Additional Information
```

Comment:
Could you add some instructions for running the tests for Nakama Unreal and Nakama Blueprints?

Reply:
I was not meant to push the "working with logs" in this PR, so I can add it next, there is already a markdown inside the Nakama Blueprints tests folder. Yes, I can also add how to run the tests in C++.